plugins/inputs/filecount/README.md

```diff
@@ -19,31 +22,41 @@ Counts files in directories that match certain criteria.
   ## Only count regular files. Defaults to true.
   regular_only = true
 
-  ## Only count files that are at least this size in bytes. If size is
-  ## a negative number, only count files that are smaller than the
-  ## absolute value of size. Defaults to 0.
-  size = 0
+  ## Only count files that are at least this size. If size is a 
+  ## negative number, only count files that are smaller than the
+  ## absolute value of size. Defaults to "0B".
+  ## For available units, see :
+  ## https://godoc.org/github.com/alecthomas/units#pkg-constants
+  size = "0B"
```

Comment:
Will an old config without the `B` qualifier still work with this change?

Reply:
No. That's a breaking change.

I see 2 options to avoid this : 

- either accept string OR int value for size, and treat string with units and ints as a byte value. But I haven't found a way to do such a thing (I tried an interface, but i'm a novice go programmer and i failed to implement it). A tip would be appreciated.
- either add a second option for handling strings with units (but what should be the name ? "size_with_units" ? and also add "recursive_print_size_with_units" ?)

Note that I'll first try to merge parts of the changes into #4752 (recursive print of matched directories).